docs/software/commands.md

```diff
@@ -167,8 +167,8 @@ debugging purpose).
         transactions are ordered by transaction fee(lower fee transactions
          are held for later).<br>
     * protocolversion (uint32) defines the protocol version to upgrade to.
-         When specified it must match the protocol version supported by the
-        node<br>
+         When specified it must match one of the protocol versions supported
+         by the node and should be greater than the current ledger<br>
```

Comment:
I think it is a typo. `It should be greater that version of the current ledger.` seems better

Reply:
I mean that the wording suggest that protocol version should be greater than the `current ledger number`, not `current ledger version`.

Reply:
ah yes, makes sense. fixed!